rcp/org.testeditor.rcp4.uatests/src/main/java/org/testeditor/fixture/swt/SWTFixture.java

```diff
@@ -110,9 +131,109 @@ public void wait(String seconds) throws NumberFormatException, InterruptedExcept
 	 */
 	@FixtureMethod
 	public boolean isViewVisible(String viewName) {
-		logger.trace("search for view with name: {}", viewName);
-		SWTBotView view = bot.viewByPartName(viewName);
+		logger.trace("search for view with titel: {}", viewName);
+		SWTBotView view = bot.viewByTitle(viewName);
 		return view.isActive();
 	}
 
+	/**
+	 * Selects an element in a tree. It takes a path string end tries to expand
+	 * the tree along the path. The node of the path is selectd.
+	 * 
+	 * @param locator
+	 *            of the tree.
+	 * @param itemName
+	 *            path to the item to be selectd. Example:
+	 *            "Root/SubNode/FinalNode"
+	 */
+	@FixtureMethod
+	public void selectElementInTreeView(String locator, String itemName) {
+		logger.trace("search for view with titel: {}", locator);
+		SWTBotTree tree = null;
+		if (locator.startsWith("[Single]")) {
```

Comment:
We need a discussion about locator types. That is adressed in https://testeditor.atlassian.net/browse/TE-350.
In this implemenation are some examples about the need for that. In the current documentation it is hieded, until we are clear about the concept.